modules/representation_management/app/models/representation_management/form_2122_base.rb

```diff
@@ -69,10 +67,9 @@ class Form2122Base
     validates :veteran_address_line2, length: { maximum: 5 }, if: -> { veteran_address_line2.present? }
     validates :veteran_city, presence: true, length: { maximum: 18 }
     validates :veteran_country, presence: true, length: { is: 2 }
-    validates :veteran_state_code, presence: true, length: { is: 2 }
-    validates :veteran_zip_code, presence: true, length: { is: 5 }, format: { with: FIVE_DIGIT_NUMBER }
-    validates :veteran_zip_code_suffix, length: { is: 4 }, format: { with: FOUR_DIGIT_NUMBER },
-                                        if: -> { veteran_zip_code_suffix.present? }
+    validates :veteran_state_code, presence: true, length: { minimum: 2 }
+    validates :veteran_zip_code, presence: true
```

Comment:
Should there at least be a minimum here?

Reply:
This makes sense, I'll put a minimum of 4 characters in a follow up PR.  As far as I can tell the shortest postal codes in the world are 4 characters.

Reply:
Minimum lengths added: https://github.com/department-of-veterans-affairs/vets-api/pull/19772/commits/4299728548f4f6a1f4313ab68e7da9f96ae095ae